packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -465,14 +467,19 @@ export default class Editor {
                     this.select(range);
                 } catch {}
             }
+        }
 
-            if (convertToDarkMode) {
-                this.convertContentToDarkMode(contentDiv, true /* skipRootElement */)();
+        // Convert content even if it hasn't changed.
+        if (convertToDarkMode) {
+            const convertFunction = this.convertContentToDarkMode(contentDiv, true /* skipRootElement */);
+            if (convertFunction) {

```

Comment:
When would this not be true? Is it when the content div has no children?

Reply:
Correct. For instance, initial calls to set content with no child elements.